zeppelin-interpreter/src/main/java/org/apache/zeppelin/conf/ZeppelinConfiguration.java

```diff
@@ -816,6 +840,13 @@ public int getClusterHeartbeatTimeout() {
     ZEPPELIN_CLUSTER_HEARTBEAT_INTERVAL("zeppelin.cluster.heartbeat.interval", 3000),
     ZEPPELIN_CLUSTER_HEARTBEAT_TIMEOUT("zeppelin.cluster.heartbeat.timeout", 9000),
 
+    ZEPPELIN_K8S_MODE("zeppelin.k8s.mode", "auto"),              // auto | on | off
```

Comment:
Later, zeppelin may also support `YARN`. Now `YARN` also supports Docker. Is it better to change this configuration to `ZEPPELIN_RUN_MODE`?
For example: ZEPPELIN_RUN_MODE("zeppelin.run.mode", "auto"),     // auto | k8s | yarn | normal